### ISSUE TITLE
Does not work in UE5.4

### ISSUE THREAD

**Issue by JZWong** [ProjectBorealis/UE4GitPlugin#155]
Currently the plugin does not work in UE5.4 because the parent class of `GitSourceControlChangelistState.h` changed the definition of the functions 
```
virtual const TArray<FSourceControlStateRef>& GetFilesStates() const override;
virtual const TArray<FSourceControlStateRef>& GetShelvedFilesStates() const override;
```

**Comment by JZWong** [comment]
For a quick fix, I had to remove the `&` from the definition and added 2 additional virtual functions for it to compile
```
virtual const TArray<FSourceControlStateRef> GetFilesStates() const override;

virtual const TArray<FSourceControlStateRef> GetShelvedFilesStates() const override;
virtual int32 GetFilesStatesNum() const override;

virtual int32 GetShelvedFilesStatesNum() const override;
```

**Comment by mastercoms** [comment]
This should be fixed with 2bd12cd31508565dc19b6478715cbe4dc527d239.
